index.html

```diff
@@ -10,23 +10,234 @@
+    </div>
+    <h3 class="connect-text">LET'S CONNECT</h3>
     <div class="socialmedia-icons">
-      <a class="twitter-icon"></a>
-      <a class="linkedin-icon"></a>
-      <a class="m-icon"></a>
-      <a class="cat-icon"></a>
-      <a class="fingers-icon"></a>
+      <ul>
+        <li class="twitter-icon"><a href=""></a></li>
+        <li class="linkedin-icon"><a href=""></a></li>
+        <li class="m-icon"><a href=""></a></li>
+        <li class="cat-icon"><a href=""></a></li>
+        <li class="fingers-icon"><a href=""></a></li>
+      </ul>
+    </div>
+    <button class="resume-access">Get my Resume</button>
+    <div>
+      <ul class="categories-info">
+        <li class="categorie" id="cat1">
+          <div class="categorie-name">
+            <h4>Languajes</h4>
+          </div>
+          <div class="arrow-img"></div>
+        </li>
+        <li class="subcategorie" id="subcat1">
+          <div class="subcat-logo"></div>
+          <div class="subcategorie-name">
+            <h4>JavaScript</h4>
+          </div>
+        </li>
+        <li class="subcategorie" id="subcat2">
+          <div class="subcat-logo"></div>
+          <div class="subcategorie-name">
+            <h4>HTML</h4>
+          </div>
+        </li>
+        <li class="subcategorie" id="subcat3">
+          <div class="subcat-logo"></div>
+          <div class="subcategorie-name">
+            <h4>CSS</h4>
+          </div>
+        </li>
+        <li class="categorie" id="cat2">
+          <div class="categorie-name">
+            <h4>Frameworks</h4>
+          </div>
+          <div class="arrow-img"></div>
+        </li>
+          <li class="separator-line">
+            <div class="line"></div>
+          </li>
+          <li class="categorie" id="cat3">
+            <div class="categorie-name">
+              <h4>Skills</h4>
+            </div>
+            <div class="arrow-img"></div>
+          </li>
+          <li class="separator-line">
+            <div class="line"></div>
+          </li>
+      </ul>
+    </div>
+  </section>
+  <section class="contactme">
+    <div class="contactme-title">
+      <h2>Contact me</h2>
+    </div>
+    <div class="contactme-intro">
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
     </div>
-  </div>
+    <form class="contactme-form" action="https://formspree.io/f/mpzbgqyj" method="post">
+      <ul class="form-elements-list">
+        <li>
+          <input class="input-name" type="text" id="user-name" placeholder="Your name" maxlength="30" required>
+        </li>
+        <li>
+          <input class="input-email" type="email" id="user-email" required placeholder="your@email">
+        </li>
+        <li>
+          <textarea class="input-textarea" required maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
Good job implementing this but the content in the form has been placed too much to the left in your design. You can correct this by adding some margin amd padding so the content is place in the center.

Your Design:
![Screenshot from 2022-07-07 11-42-48](https://user-images.githubusercontent.com/79506105/177724954-5d40fa2b-d0ef-423f-b637-df7ac9af11ee.png)
I would advise you carefully follow the template on figma.

Please follow this design: 
![Screenshot from 2022-07-07 11-45-41](https://user-images.githubusercontent.com/79506105/177725180-2653ab5a-0a49-4fe0-a434-d28ed7d1df83.png)

Reply:
Hello KayLemba:

I appreciate the feedback. After making some changes I suggest you review my project setting the mobile screen width to 375px which is the Figma set width for the mobile version. My project might look different to Figma for wider screen sizes since I made it responsive to different screen sizes according to another feedback that I received before where they encouraged me to set the widths using percentages % to make it more responsive to screen size changes.